client/my-sites/checkout/checkout-thank-you/business-plan-details.jsx

```diff
@@ -49,6 +51,32 @@ const BusinessPlanDetails = ( {
 		<div>
 			{ googleAppsWasPurchased && <GoogleAppsDetails isRequired /> }
 
+			{ isEnabled( 'jetpack/features-section' ) && (
+				<>
+					<PurchaseDetail
+						icon={ <img alt="" src={ jetpackBackupImage } /> }
+						title={ i18n.translate( 'Check your backups' ) }
+						description={ i18n.translate(
+							'Backup gives you granular control over your site, with the ability to restore it to any previous state, and export it at any time.'
+						) }
+						buttonText={ i18n.translate( 'See the latest backup' ) }
+						href={ `/backup/${ selectedSite.slug }` }
+						onClick={ trackOnboardingButtonClick }
+					/>
+
+					<PurchaseDetail
+						icon={ <img alt="" src={ jetpackScanImage } /> }
+						title={ i18n.translate( 'Scan your site' ) }
+						description={ i18n.translate(
+							'Scan gives you automated scanning and one-click fixes to keep your site ahead of security threats.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Jetpack Scan uses automated scanning and one-click fixes to keep your site ahead of security threats.' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/backup-scan-business-congratulation)